plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -9,6 +10,11 @@ import Pod from '../../structs/Pod';
 import Service from '../../structs/Service';
 import ServiceTree from '../../structs/ServiceTree';
 
+const REDIRECT_DELAY = 500;
```

Comment:
This feels a bit random. It's a good idea to comment what's the reasoning behind this number.